### PR TITLE
Add sf-rest connector overview

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Salesforce REST Connector Overview
+
+This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links of the shown bellow to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md:3]:
```suggestion
This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links below to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
```
Shall we replace the first sentence with some information about the following:
- API(s) that we are connecting to.
- The connectors we have.

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Salesforce REST Connector
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md:5]: Add some information about what exactly can be done using this connector.

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* **[Salesforce Rest API Connector Example](sf-rest-connector-example.md)**
+
+  This example explains how to use the Salesforce client to connect with the Salesforce instance and perform the **create** and **retrive** operations.
+
+* **[Salesforce Rest API Connector Configuration](sf-rest-connector-config.md)**
+
+  This documentation provides a reference guide for the Salesforce Rest API operations.
+  
+## Salesforce Inbound Endpoint
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md:19]: Add some information about what can be done via the endpoint.

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md]: Updated.

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md:25]: Added.

File: en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md:9]: Added.

**Pull request merged**